icu4j/maven-build/maven-icu4j-localespi/pom.xml

```diff
@@ -0,0 +1,242 @@
+<?xml version="1.0"?>
+<!--
+* © 2016 and later: Unicode, Inc. and others.
```

Comment:
This is a new file, the copyright should be 2022.

True for all files in icu4j-maven-build (everything listed as added by git)